luigi/interface.py

```diff
@@ -122,6 +122,10 @@ class core(task.Config):
     parallel_scheduling = parameter.BoolParameter(
         default=False,
         description='Use multiprocessing to do scheduling in parallel.')
+    parallel_scheduling_processes = parameter.IntParameter(
+        default=None,
```

Comment:
I think this will cause a warning, can you set the default to be the value to be zero (`0`)?